### PR TITLE
Add user stories markdown

### PR DESCRIPTION
**Pull request by dietbrand**
Writing user stories is the first step. I'd like to refine these into actual tasks and update the PR so it has flow charts or clear definitions of what the tasks are. I will definitely appreciate feedback on this @camelopardalus 😄

### CODE REVIEW

**Comment by dietbrand** [comment]
More information about writing user stories: https://www.atlassian.com/agile/project-management/user-stories. Template is lifted from there.

File: design/userstories.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+As a user, I want to be able to add a book, so my library reflects the real world
+
+As a user, I want to keep track of the books my friends have borrowed, so I can manage my possessions
+
+As a user, I want to sort my books, so that I can see recent additions
+
+As a user, I want to sort my books, so that I can see all books from one author
+
+As a user, I want to group my books, so that I can see all books from a series or publisher
```
Comment (tluyckx) [inline design/userstories.md:21]: By series, do you mean a series of books written by the same author (like a trilogy), or a series of books by the same publisher (like the Little Black Classics by Penguin Books)?

File: design/userstories.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### Template
+
+“As a [persona], I [want to], [so that].”
+
+## General
+
+As a user, I want to see all my books, so that I can have an overview
+
+As a user, I want to be able to view a book, so I can see more information
```
Comment (tluyckx) [inline design/userstories.md]: Maybe change 'view' to 'select' for this one

**Review by tluyckx — COMMENTED** [review]
Good work. I'll add some user stories too during the next few days.

**Comment by tluyckx** [comment]
This could be useful too in the next step(s): https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

File: design/userstories.md (hunk 1 above)
Comment (dietbrand) [reply design/userstories.md:21]: I was thinking of a general grouping functionality based on the attribute you wanted, like author, series or publisher or even year. Those sections are then sorted based on the active sort.

**Review by tluyckx — APPROVED** [review]

**Pull request merged**